docs/usage/using-static-ip.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 3
+---
+# About huronOS instance Static IP configuration
+This feature allows you to assign each usb a static IP address, this way, you will not have to assign an IP address each time an instance has booted up if a DHCP server is not available/enabled. 
+
+The current implementation is expected to work with too little issues, yet since testing is one of the things we lack the most, we're not sure on how smooth the use of this feature is going to be.
+
+# How to use a static IP on an instance
+
+To use static IP, you need to assign the instance ip address, ip gateway and network mask at the time of using the installer, for example `sudo ./install.sh -ia 192.168.1.2 -ig 192.168.1.254 -nm`
+
+# Notes
+
+- This feature only works on ethernet interfaces (since wifi connections are more likely to provides an IP, whereas this feature was born from the lack of dynamic ip leasing in one institution that uses HuronOS)
+
+- If the computer has multiple network interfaces it will try on each available interface in no specific order until the configuration gets internet output through said interface, if it had no internet that interface would go back to dhcp
+
+- If the network mask is different from the one
+
+- If the static ip failed to be assigned to a network interface of the computer, everything would still be the same, that is, every interface would have dhcp enabled
+
+# Troubleshooting
+
+What would be good to add here?
```

Comment:
Just put in the shoes of a sysadmin trying to setup the system on different scenarios. What do you think they can do wrong?
- Bad configured network mask, what should they do? How do they know this is the wrong part?
- What if the IP is already taken, how do we notify them? What's going to be noticeable for the admin to know something is wrong?
- What happens if the IP is malformed or invalid?
- What happens if the admin didn't set a directives URL?
- If the directives URL is malformed the system won't set the static IP, how will the sysadmin know that the directives URL is the problem and no the IP config?

This are only some of the ideas that come to my mind. You can find out more by going out of the dev perspective and try to think about all the possible scenarios that the script might encounter.

Reply:
Okay